packages/teuchos/comm/src/Teuchos_TimeMonitor.cpp

```diff
@@ -1488,7 +1488,7 @@ namespace Teuchos {
     stackedTimer_ = t;
   }
 
-  const Teuchos::RCP<Teuchos::StackedTimer>&
+  Teuchos::RCP<Teuchos::StackedTimer>
```

Comment:
Yes.  Thank you!  See "return smart pointers by value" in section "5.8.5 Idioms for returning objects from C++ functions" in [Teuchos MM document](https://trilinos.org/docs/dev/packages/teuchos/doc/html/TeuchosMemoryManagementSAND.pdf).

Reply:
@bartlettroscoe you're welcome :-)